custom_mutators/libfuzzer/FuzzerIO.cpp

```diff
@@ -83,6 +83,8 @@ void WriteToFile(const std::string &Data, const std::string &Path) {
 
 void WriteToFile(const uint8_t *Data, size_t Size, const std::string &Path) {
 
+  return;
+ 
   // Use raw C interface because this function may be called from a sig handler.
```

Comment:
Do you (Marc) want to keep the dead code?

Reply:
yes :)